README.md

```diff
@@ -259,12 +259,13 @@ rm "$PREFIX/bin/ani-cli"
 - sed
 - awk
 - curl
+- wget
 - openssl
 - mpv - Video Player
 - iina - mpv replacement for MacOS
-- axel - Download manager
+- aria2c - Download manager
```

Comment:
In favor of that revert.
`aria2c` is useful outside of ani-cli as a torrent client, unlike `axel`.